tests/framework/db/QueryBuilderTest.php

```diff
@@ -928,6 +928,61 @@ public function columnTypes()
                     'cubrid' => "int NOT NULL AUTO_INCREMENT PRIMARY KEY COMMENT 'test comment'",
                 ],
             ],
+            [
+                Schema::TYPE_PK . " FIRST",
```

Comment:
Hmm, I think then with Oracle, adding a position modifier will have no effect, and thus the tests are correct. It is the same as Postgres in that respect.